jobs/config.json

```diff
@@ -10596,7 +10592,7 @@
       "--ginkgo-parallel",
```

Comment:
(two lines above, remove the `pull-kubernetes-e2e-kops-aws-prow.env` reference?)